test/libsolidity/syntaxTests/userDefinedValueType/non_value_type_function_err.sol

```diff
@@ -0,0 +1,3 @@
+type MyFunction is function(uint) returns (uint);
+// ----
+// TypeError 8657: (19-49): The underlying type for a user defined value type has to be an elementary value type.
```

Comment:
Since we now generate the wrapping and unwrapping functions internally, is there any reason for this not to be allowed? But also fine to keep disallowed for now.